x-pack/legacy/plugins/siem/public/components/page/hosts/hosts_table/columns.tsx

```diff
@@ -56,10 +56,29 @@ export const getHostsColumns = (
                 </DragEffects>
               ) : (
                 <AddToKql
+                  id="global"
                   indexPattern={indexPattern}
-                  expression={`host.name: ${escapeQueryValue(hostName[0])}`}
-                  componentFilterType="hosts"
-                  type={type}
+                  filter={{
+                    meta: {
+                      alias: null,
+                      negate: false,
+                      disabled: false,
+                      type: 'phrase',
+                      key: 'host.name',
+                      value: hostName[0] as string,
+                      params: {
+                        query: hostName[0],
+                      },
+                    },
+                    query: {
+                      match: {
+                        'host.name': {
+                          query: hostName[0],
+                          type: 'phrase',
+                        },
+                      },
+                    },
+                  }}
```

Comment:
A helper for these would be beneficial and clean things up a bit/make it easier to use `AddToKql` elsewhere.